addons/website_sale/models/sale_order_line.py

```diff
@@ -46,6 +46,7 @@ def _get_sale_order_line_multiline_description_sale(self):
         return description
 
     def get_description_following_lines(self):
+        self._compute_name()
```

Comment:
Hi @barracudapps 

If we go with changing this in stable, I'd rather update the order each time customer change the language rather than each time `get_description_following_lines` is called